src/client/utils/dom.js

```diff
@@ -685,6 +689,10 @@ export function isTableDataCellElement (el) {
     return instanceToString(el) === NATIVE_TABLE_CELL_STR;
 }
 
+export function hasElementTextProperty (el) {
+    return isScriptElement(el) || isAnchorElement(el) || isOptionElement(el) || isFormElement(el) || isTitleElement(el);
+}
```

Comment:
because `title` may have `<script>` inside and it's valid markup, so we need clean up its `text`